keep/providers/wazuh_provider/custom-keep.py

```diff
@@ -0,0 +1,171 @@
+import json
+import os
+import sys
+from datetime import datetime, timezone
```

Comment:
Please explicitly comment both files that they are not going to be executed on Keep side and are stored here to be served from the GitHub to be used inside Wazuh.